design-system/materials/internals/definition.yaml

```diff
@@ -757,6 +789,13 @@ decisionGroupsByTheme:
         font-weight-for-button:
           choice: font-weight-500
 
+    fontColors:
+      label: Font Colors
+      prefix: font-color
+      decisions:
+        font-color-for-input-when-readonly:
+          choice: color-neutral-60
```

Comment:
Shouldn't this value be `color-neutral-40`?

Reply:
Updated [38d3d54](https://github.com/commercetools/ui-kit/pull/2347/commits/38d3d543be3051df492edcd0d5722413e61a5499)